src/databricks/labs/ucx/workspace_access/groups.py

```diff
@@ -975,6 +910,81 @@ def _get_strategy(
         )
 
 
+class AccountGroupLookup:
+    def __init__(self, ws: WorkspaceClient):
+        self._ws = ws
+
+    def pick_owner_group(self, prompt: Prompts) -> str | None:
```

Comment:
Prefer to not have the prompts in classes which are "deeper" into ucx

Reply:
Would the following make more sense:

```
def list(self, *, user: User | None = None) -> list[Group]
    """List account groups.
    
    Args:
        user (User | None ) : Filter for groups that contain the given user.
    """

Reply:
Maybe make user none optional if that suits you better

Reply:
That's a directive from the boss to reduce code in cli.

Reply:
hmm oke, could still be an inbetween option, but not going to block the PR over it